content/influxdb/cloud/organizations/migrate-org.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Migrate organization
+seotitle: Migrate organization in InfluxDB
+description: Replicate the state of an organization in InfluxDB
+menu:
+  influxdb_cloud:
+    name: Migrate organization
+    parent: Manage organizations
+weight: 110
+---
+
+To duplicate the state of an organization:
```

Comment:
I think a quick summary explanation would help set context for the following steps. E.g. an org's state consists of both metadata resources (dashboards, buckets, etc) and the data (time-series).